app/controllers/declaration/AdditionalActorsSummaryController.scala

```diff
@@ -58,6 +58,8 @@ class AdditionalActorsSummaryController @Inject()(
         (formWithErrors: Form[YesNoAnswer]) => BadRequest(additionalActorsPage(mode, formWithErrors, actors)),
         validYesNo =>
           validYesNo.answer match {
+            case YesNoAnswers.yes if (mode.isErrorFix) =>
+              navigator.continueTo(mode, routes.AdditionalActorsAddController.displayPage, mode.isErrorFix)
```

Comment:
Do you need  this extra case statement? You can pass `mode.isErrorFix` to the `continueTo` method even if its value is false (that is the parameter's default value anyway) :slightly_smiling_face: